src/web/nextui/src/app/components/MarkdownRenderer.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactMarkdown from 'react-markdown';
+import CustomImage from './CustomImage';
+
+interface MarkdownRendererProps {
+  content: string;
+}
+
+const MarkdownRenderer: React.FC<MarkdownRendererProps> = ({ content }) => {
+  return (
+    <ReactMarkdown
+      components={{
+        img: ({ node, ...props }) => (
+          <CustomImage src={props.src || ''} alt={props.alt || ''} {...props} />
+        ),
```

Comment:
This prop is used to override the default renderers for different types of Markdown nodes. In this case, it's overriding the renderer for image nodes (img). For each image node, it's rendering a `CustomImage` component instead of the default `img` element.